index.html

```diff
@@ -36,3 +36,136 @@ <h3 class="intro">I'm Bunbee<br>Glad to see<br> you!</h3>
+           </ul>
+        </div>
+        <button class="btn1">See Project</button>
+    </section>
+    <section class="card-section">
+        <div class="card-image">
+            <img src="img/SnapshootPortfolio(2).png" alt="firsroject">
+        </div>
+        <p class="card-header">Multi-Post<br> Stories</p>
+        <ul class="card-links">
+            <li>CANOPY</li>
+            <li> Black End Dev</li>
+            <li>2015</li>
+        </ul>
+        <p class="card-text">A daily selection of privately <br>personalised reads; no accounts or sign-ups required</p>
+        <div class="lang-section">
+           <ul class="card-lang">
+            <li>html</li>
+            <li>css</li>
+            <li>javaScript</li>
+           </ul>
+        </div>
+        <button class="btn1">See Project</button>
+    </section>
+    <section class="card-section">
+        <div class="card-image">
+            <img src="img/SnapshootPortfolio(3).png" alt="firsproject">
+        </div>
+        <p class="card-header">Tonic</p>
+        <ul class="card-links">
+            <li>CANOPY</li>
+            <li> Black End Dev</li>
+            <li>2015</li>
+        </ul>
+        <p class="card-text">A daily selection of privately <br>personalised reads; no accounts or sign-ups required</p>
+        <div class="lang-section">
+           <ul class="card-lang">
+            <li>html</li>
+            <li>css</li>
+            <li>javaScript</li>
+           </ul>
+        </div>
+        <button class="btn1">See Project</button>
+    </section>
+    <section class="card-section">
+        <div class="card-image">
+            <img src="img/SnapshootPortfolio(4).png" alt="firstoject">
+        </div>
+        <p class="card-header">Multi-Post<br>Stories</p>
+        <ul class="card-links">
+            <li>CANOPY</li>
+            <li> Black End Dev</li>
+            <li>2015</li>
+        </ul>
+        <p class="card-text">A daily selection of privately <br>personalised reads; no accounts or sign-ups required</p>
+        <div class="lang-section">
+           <ul class="card-lang">
+            <li>html</li>
+            <li>css</li>
+            <li>javaScript</li>
+           </ul>
+        </div>
+        <button class="btn1">See Project</button>
+    </section>
+
+
+    <section class="about-me">
+        <h1 class="about-header">About<br> Myself</h1>
+        <p class="about-para"> Hello I'm a software devoloper! I can help you build a product, feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded, dont hesitate to contact me.</p>
+        <p class="connect">LET'S CONNECT</p>
+        <ul class="social-media">
+            <li class="fa fa-twitter fa-2x"></li>
+            <li class="fa fa-linkedin fa-2x"></li>
+            <li class="fa fa-medium fa-2x"></li>
+            <li class="fa fa-github fa-2x"></li>
+            <li class="fa fa-facebook fa-2x"></li>
+           </ul>
```

Comment:
Icons have a very small spacing between them. Please see the design and give them a bit more space.